workshops/migrations/0012_auto_20150612_0807.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+TRANSLATE_NAMES = {
+    'Git': ['swc/git-novice'],
+    'Make': [],
```

Comment:
On Tue, Jun 16, 2015 at 10:06:16AM -0700, Greg Wilson wrote:
> > +TRANSLATE_NAMES = {
> > +    'Git': ['swc/git-novice'],
> > +    'Make': [],
> 
> Not in a usable state - Mike Jackson has just offered us a rework
> that's much more usable, but it isn't ready for prime time yet.

For example, I'm willing to teach the Make lesson however it gets
written (or to just teach Make using a one-off lesson based on how
I've taught Make in the past).  Do we want to use the WIP lesson as a
place-holder for folks like me?  Or do we want to re-populate that
qualification once we have an official Make lesson?